docs/setup/install/install-devtron-local.md

```diff
@@ -0,0 +1,75 @@
+# Install Devtron on Minikube, K3s, Kind 
+You can install and try Devtron with CICD integration on a high-end Laptop or a PC, but the laptop/PC may start to respond slowly, so it is recommended to uninstall Devtron from your laptop before shutting it down.
```

Comment:
Make it more generic, people can also install microk8s on a highend cloud server.

Reply:
added Microk8s cluster for Cloud VM